Marlin/src/gcode/motion/M290.cpp

```diff
@@ -44,13 +44,15 @@
     ) {
       probe_offset[Z_AXIS] += offs;
       SERIAL_ECHO_START();
-      SERIAL_ECHOLNPAIR(MSG_PROBE_OFFSET MSG_Z ": ", probe_offset[Z_AXIS]);
+      SERIAL_ECHOPGM(MSG_PROBE_OFFSET);
+      SERIAL_ECHOLNPAIR(MSG_Z ": ", probe_offset[Z_AXIS]);
     }
     #if ENABLED(BABYSTEP_HOTEND_Z_OFFSET)
       else {
         hotend_offset[Z_AXIS][active_extruder] -= offs;
         SERIAL_ECHO_START();
-        SERIAL_ECHOLNPAIR(MSG_Z_OFFSET ": ", hotend_offset[Z_AXIS][active_extruder]);
+        SERIAL_ECHOPGM(MSG_Z_OFFSET);
+        SERIAL_ECHOLNPAIR(": ", hotend_offset[Z_AXIS][active_extruder]);
```

Comment:
Ah, I see the problem. This is supposed to be the non-translatable string `MSG_PROBE_OFFSET`, not `MSG_Z_OFFSET`. So the serial macro can remain unchanged.